.travis.yml

```diff
@@ -16,45 +16,30 @@ addons:
       - xz
       - yajl
 
+env:
+  - VIRTBLOCKS="golang"
+  - VIRTBLOCKS="rust"
```

Comment:
I don't think this works the way you expected it to work: see https://travis-ci.org/virtblocks/libvirt/builds/577328929 for evidence.

I'd say just keep it simple, by appending another macOS stanza like the existing one but with s/rust/golang/, and leave the (optional) cleanup for a separate PR.